shared/util.go

```diff
@@ -1237,3 +1237,13 @@ func InSnap() bool {
 
 	return false
 }
+
+// JoinUrlPath return the join of the input urls/paths sanitized.
+func JoinUrls(baseUrl, path string) (string, error) {
+	u, err := url.Parse(baseUrl)
+	if err != nil {
+		return "", err
+	}
+	u.Path = filepath.Join(u.Path, path)
```

Comment:
You wouldn't use this function as it would use a backslash rather than slash on Windows

Reply:
Yeah, i thought to test after I already pushed. I will add it.
Instead, bleah for Windows :P 
Does filepath take care of backslash/slash for Windows too? I never investigated on this, it's about 10 years that I don't use a Microsoft system :D

Reply:
Yeah, pretty sure it will be backslash on Windows. You could just add the test with no other changes, we run tests for `shared/..` on Windows too, so Github would let you know pretty quick :)

Reply:
Add a reference to a golang issue about this argument: https://github.com/golang/go/issues/30616

Maybe, only `path.Join(..)` works.